utils/debug/Debug.js

```diff
@@ -95,6 +108,73 @@ function Debug(view, datDebugTool, chartDivContainer) {
         // debug graphs update
         debugChartUpdate(duration);
     };
+
+    // Camera debug
+    const helper = new CameraHelper(view.camera.camera3D);
+    const debugCamera = view.camera.camera3D.clone();
+    debugCamera.fov *= 1.5;
+    debugCamera.updateProjectionMatrix();
+    const g = view.mainLoop.gfxEngine;
+    const r = g.renderer;
+    let fogDistance = view.fogDistance;
+    helper.visible = false;
+    view.scene.add(helper);
+
+    function tileDisplay(obj) {
+        if (obj.setFog && fogDistance) {
+            obj.setFog(fogDistance);
+        }
+    }
+
+    view.render = function render() {
```

Comment:
You should instead use the `MAIN_LOOP_EVENTS.AFTER_RENDER` mechanism - it's made for this.